react-ui/src/project/index.js

```diff
@@ -164,6 +164,19 @@ const Project = () => {
                                 </div>
                             </div>) : null}
 
+                            {isCycleTime ? (<div style={{ display: 'flex', flexDirection: 'column', gap: '10px', marginBottom: '20px' }}>
```

Comment:
Can't we just check **isCycleTime** where we are checking **isLeadTime** using or condition?
https://github.com/sverma89asu/SER516-Team-Cleveland/blob/9976f98c4739f3680de9f646db4752b796b88ff2/react-ui/src/project/index.js#L154

Reply:
Modified